File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
import sys 
input = sys.stdin.readline
sys.setrecursionlimit(10**6)
tc= int(input())
for _ in range(tc):
    H,W,N = map(int, input().split())
    floor = 0
    room = 0
    if N % H == 0:
        floor = H * 100
        room = N // H
    else:
        floor = (N % H) * 100
        room = 1 + N // H
    print(floor + room)

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

var (
	reader = bufio.NewReader(os.Stdin)
	writer = bufio.NewWriter(os.Stdout)
)

func main() {
	defer writer.Flush()
	var tc int
	var h, w, n, floor, room int
	fmt.Fscanf(reader, "%d\n", &tc)
	for i := 0; i < tc; i++ {
		fmt.Fscanf(reader, "%d %d %d\n", &h, &w, &n)
		floor = 0
		room = 0
		if n%h == 0 {
			floor = h * 100
			room = int(n / h)
		} else {
			floor = (n % h) * 100
			room = 1 + n/h
		}
		fmt.Fprintf(writer, "%d\n", room+floor)
	}
}
